src/main/java/uk/ac/ebi/ena/webin/cli/service/handler/DefaultErrorHander.java

```diff
@@ -1,48 +0,0 @@
-/*
```

Comment:
I think we should continue to use s NotFoundErrorHandler (and DefaultErrorHander if needed). See previous comment.